tests/sagemaker/mock/test_sagemaker_service_mock.py

```diff
@@ -3,16 +3,10 @@
 import boto3
 import pytest
 
+from tests.helper_functions import set_boto_credentials
```

Comment:
Is this here to be imported elsewhere as tests.sagemaker.mock.set_boto_credentials? Do we use it?

Reply:
This fixture is needed for the test suite in which it's imported. This suite tests the behavior of the SageMaker mocks

Reply:
ah somehow i read this as a utils module not an actual test module.